kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/TopicsController.java

```diff
@@ -38,6 +38,12 @@
         .switchIfEmpty(Mono.just(ResponseEntity.notFound().build()));
   }
 
+  @Override
+  public Mono<ResponseEntity<TopicDTO>> recreateTopic(String clusterName,
+                                                      String topicName, ServerWebExchange serverWebExchange) {
+    return topicsService.recreateTopic(getCluster(clusterName), topicName).map(ResponseEntity::ok);
```

Comment:
according to swagger doc status should be 201, not 200. Either edit it here or change doc

Reply:
Fixed.